modules/installing-gcp-cluster-creation.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+// * installing-gcp-user-defined-labels-and-tags.adoc
+
+:_content-type: PROCEDURE
+[id="installing-gcp-cluster-creation_{context}"]
+= Cluster creation phase 
+
+After you successfully install an add-on service on your {product-title} cluster, you can access the service by using the GCP.
```

Comment:
Should this be a prerequisite?

Reply:
@bharath-b-rh can you ptal and confirm if I should add this as a prerequisite?

Reply:
@subhtk 
```
After you successfully install an add-on service on your {product-title} cluster, you can access the service by using the GCP.

.Prerequisites

* You have successfully installed a service on your {product-title} cluster.
```
This is not related to GCP tags or labels, can be removed.

Reply:
@bharath-b-rh Done. Can you let me know if there is anything thta needs to be added here in the procedure?

Reply:
@subhtk Current content describing, configuring tags and labels in install-config captures relevant info, I think nothing more is required.

Reply:
As per Bharat's comment, this needs to be removed.

Reply:
As suggested, this part is removed 

"After you successfully install an add-on service on your {product-title} cluster, you can access the service by using the GCP.

.Prerequisites

* You have successfully installed a service on your {product-title} cluster."